app/src/lib/progress/from-process.ts

```diff
@@ -23,8 +23,13 @@ export async function executionOptionsWithProgress(
   let lfsProgressPath = null
   let env = {}
   if (options.trackLFSProgress) {
-    lfsProgressPath = await createLFSProgressFile()
-    env = { GIT_LFS_PROGRESS: lfsProgressPath }
+    try {
+      lfsProgressPath = await createLFSProgressFile()
+      env = { GIT_LFS_PROGRESS: lfsProgressPath }
+    } catch (e) {
+      log.error('Error writing LFS progress file:', e)
```

Comment:
Could we drop the trailing `:` in the error message please? I know we've been a bit inconsistent here but when you're passing an error the message should stand by itself. The logging framework is responsible for the formatting of the error + message.